junebug/api.py

```diff
@@ -301,3 +304,33 @@ def get_message_status(self, request, channel_id, message_id):
     @app.route('/health', methods=['GET'])
     def health_status(self, request):
         return response(request, 'health ok', {})
+
+    @app.route('/channels_health', methods=['GET'])
+    @inlineCallbacks
+    def channels_health(self, request):
```

Comment:
We should probably put this in with the normal channel health, rather than having a separate endpoint for it.

This should also be behind a config flag that is by default disabled, and have documentation around if you're running rabbitmq, then you can enable this to get this extra functionality